phoenix-core/src/main/java/org/apache/phoenix/index/GlobalIndexChecker.java

```diff
@@ -315,7 +320,7 @@ private boolean verifyRowAndRemoveEmptyColumn(List<Cell> cellList) throws IOExce
                     // we consider anything that is not UNVERIFIED means VERIFIED. IndexTool should be used to
```

Comment:
These comment needs to be changed too to reflect new behavior.

Reply:
Will remove the comments

Reply:
Done